src/main/java/com/mathworks/ci/freestyle/RunMatlabTestsBuilder.java

```diff
@@ -48,16 +48,22 @@ public class RunMatlabTestsBuilder extends Builder implements SimpleBuildStep {
     private transient boolean tapChkBx;
     private transient boolean junitChkBx;
     private transient boolean coberturaChkBx;
+    private transient boolean htmlChkBx;
     private transient boolean stmResultsChkBx;
     private transient boolean modelCoverageChkBx;
+    private transient boolean modelCoverageChkBxHTML;

```

Comment:
Looking at other variable names, using the following seems consistent:

`modelCoverageChkBx` -> `coberturaModelCoverageChkBx`
`modelCoverageChkBxHTML` -> `htmlModelCoverageChkBx`